docs/getting-started/explore-intro.asciidoc

```diff
@@ -2,7 +2,7 @@
 
 = Explore
 
-The following section includes an overview of the *Hosts*, *Network*, and *Users* pages.
+The following section includes an overview of the *Hosts*, *Network*, *Users*, and *Threat Intelligence* pages.
```

Comment:
I noticed the same thing that @nastasha-solomon commented about the Threat Intelligence page being absent from the latest builds (my latest shared server is BC3). Should it be removed from this line?